native/more/add-friends-modal.react.js

```diff
@@ -0,0 +1,309 @@
+// @flow
+
+import React from 'react';
+import { Text, View, ActivityIndicator, Alert } from 'react-native';
+import { createSelector } from 'reselect';
+import invariant from 'invariant';
+
+import type { RootNavigationProp } from '../navigation/root-navigator.react';
```

Comment:
Nit - I usually separate imports into four ordered categories:
1. Types
2. Imports from outside monorepo
3. Imports from other packages inside monorepo (usually `lib`)
4. Imports from the local package

Open to changing this, though. What do you think? Alternately, we could just set up Prettier/ESLint to auto-alphabetize the imports or something

Reply:
Good question. So it's worth noting that since we're using `babel-eslint`, any `import type` statements get stripped before ESLint sees them. So there are some examples of duplicate `import`s where one is just `import type`.

I don't have a strong opinion on how to handle types vs. normal imports, you'll see me do it in both ways across the codebase. I'm open to whatever your preference is, we can standardize on that. But in general I think my approach has been - if it's a small type that is connected to the import, then do it on the same line. But if there are a lot of type imports, or if the type imports aren't necessarily related to the other imports, I sometimes put them on a separate line.

Reply:
The way you have been doing it so far is ok for me